### ISSUE TITLE
Saving output in JASP

### ISSUE THREAD

**Issue by joucoski** [jasp-stats/jasp-desktop#2208]
I can not save the output in any useful format since the HTML format that is generated is very restrictive. In the output window, by clicking Copy in the drop-down list, you can not paste into Notepad. I copied to LibreOffice Calc (free version of a spreadsheet) but did not copy into a usable format. In the matrix each row shows the name of the variable and then with many spaces up to the value of interest. Then the next line has the variable name and other hundreds of spaces up to the correlation value. Is there no other way to save the output? Or could the version to be copied to LibreOffice Calc  be improved so that these unnecessary spaces do not appear?

![captura de tela de 2017-12-06 16-46-04](https://user-images.githubusercontent.com/14141667/33679166-19a611a2-daa5-11e7-8233-f50e51ca13e4.png)

**Comment by EJWagenmakers** [comment]
Yes this is annoying. I think the format includes particular table information that does not copy over nicely (or not at all) to a program like Notepad. It does copy the tables to, say, Word or LibreOffice Writer. It would be good to be able to copy the tables as "text only" (so it has hyphens instead of horizontal lines) or even LaTeX code (a man can dream).
@FransMeerhoff @boutinb

**Comment by benediktclaus** [comment]
Table output in Latex code is highly requested. That would be an awesome feature!
